pinot-common/src/main/java/org/apache/pinot/common/utils/FileUtils.java

```diff
@@ -18,16 +18,22 @@
  */
 package org.apache.pinot.common.utils;
 
+import java.io.Closeable;
 import java.io.File;
 import java.io.IOException;
 import java.nio.channels.FileChannel;
+import java.util.Arrays;
 import java.util.Random;
+import javax.annotation.Nonnull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 public class FileUtils {
   private FileUtils() {
   }
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(FileUtils.class);
```

Comment:
Remove this line as logger is never used

Reply:
Done